nio_cli/commands/config.py

```diff
@@ -29,18 +33,23 @@ def config_project(name='.'):
     os.remove(conf_location)
     os.rename(tmp.name, conf_location)
 
-    secure = input('Optional secure instance configuration [Y/N]: ')
+    secure = input('Secure instance with SSL (optional) [y/n, default = n]: ')
```

Comment:
The way I normally see this notated would be like this: `[y/N]`

Reply:
I agree, and that's the way I would normally have done it, but I wanted to lower the bar a little for people who might not be familiar with that... I can make the default capital, but I'd still want to explicitly state the default so as not to have people typing capital letters.

Who knows... maybe we'll send them out into the world a little better prepared!

Reply:
this is done